pkg/admission/webhookconfig.go

```diff
@@ -0,0 +1,66 @@
+package admission
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes"
+)
+
+// PatchCABundleOnValidatingWebhooks patches the CA bundle of all validating webhook configurations that have the specified labels in the cluster.
+// Webhook configurations should have the following labels:
+// "grafana.com/inject-rollout-operator-ca": "true",
+// "grafana.com/namespace":                  <specified namespace>,
+func PatchCABundleOnValidatingWebhooks(ctx context.Context, logger log.Logger, kubeClient kubernetes.Interface, namespace string, caPEM []byte) error {
```

Comment:
Doesn't this belong more in the tlscert package?

Reply:
I don't have a strong opinion, moved in https://github.com/grafana/rollout-operator/pull/25/commits/2a8c86a064c2d6dffcbb34276740473f569770bc